config/jobs/kubernetes/sig-arch/kubernetes-depstat.yaml

```diff
@@ -0,0 +1,28 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: kubernetes-depstat
+    decorate: true
+    always_run: true
+    skip_report: true
+    spec:
+      containers:
+      - image: golang
+        command: ["/bin/bash"]
+        args:
+        - -c
+        - |
+          set -euo \
+          BASE_SHA="${PULL_BASE_SHA:-""}" \
+          ARTIFACTS="${ARTIFACTS:-/tmp/artifacts}" \
+          mkdir -p "${ARTIFACTS}" \
+          cd "$ARTIFACTS" \
+          go install github.com/kubernetes-sigs/depstat@latest \
+          cd /go/src/k8s.io/kubernetes \
+          function write_report() { depstat stats --json; } \
+          function diff_reports() { diff "$1" "$2"; } \
+          write_report > "${ARTIFACTS}/stats.json" \
+          if [ -n "${BASE_SHA}" ]; then \
+          git checkout -b base "${BASE_SHA}" \
+          write_report > "${ARTIFACTS}/stats-base.json" \
+          diff_reports "${ARTIFACTS}"/stats-base.json "${ARTIFACTS}"/stats.json \
+          fi \
```

Comment:
remove `\` at the end of the last line.

Reply:
there are other issues as well. here's how you test

![image](https://user-images.githubusercontent.com/23304/118120494-fe34dc00-b3bd-11eb-93f8-1341f48e7bc1.png)

Reply:
you may need to separate lines using `;`